CONTRIBUTING.md

```diff
@@ -48,13 +48,18 @@ Merging Pull Requests (for project contributors with write access)
   - add `JAVA_HOME` to environment variables
 - Install [Maven](http://maven.apache.org/download.cgi)
   - add `MAVEN_HOME` to environment variables
-
+- Install the SDK Build Tools Maven artifact by running from the root folder:
+  ```bash
+  mvn -f eng/code-quality-reports/pom.xml install
```

Comment:
It would be better to update the `Building and Testing` section with the following command:

`mvn -f pom.client.xml -Dgpg.skip -DskipTests -Dinclude-non-shipping-modules clean install `

`-Dinclude-non-shipping-modules` includes all the necessary build tools and this option is necessary only when building for the first time or whenever there are any changes to the build tools configuration.

Reply:
This is done with my recent changes.